config.json

```diff
@@ -17,7 +17,7 @@
 	"session_refresh_time": 18000,
 	"localOnly": false,
 	"enableLocalMode": true,
-	"localModeSocketLocation": "/var/tmp/focalboard_local.socket",
+	"localModeSocketLocation": "./focalboard_local.socket",
```

Comment:
This change is changing the behavior on all the systems, can we make it change only for windows? For example, on windows startup if this variable is `/var/tmp/focalboard_local.socket` (the default value), transform that to something else, for example `./focalboard_local.socket`, or the way Windows store temporary files.

Reply:
Well, we could do that, but that would involve adding a lot of Code specifics for Windows that only checks if the Default Config isn't Overridden. 

And What happens on the Very rare Use case when a Windows User Actually wants to use "/var/tmp/focalboard_local.socket"?
The Error could also appear on another (Linux) System where "/var/tmp" was deleted or simply doesn't exist.
With the Current Implementation it is always made Sure that the Path Location Exists.
(And it always gets Deleted with the Folder, and doesn't Stay behind if the User Decides to do so on Every System)
And if it is really wanted, the Config can Still be Changed to Every Path the User Wants.

I personally wouldn't Implement a Windows Specific Way, as this adds a lot of additional trouble that is only Relevant for one specific System.

But I am not Qualified to make that Decision :)
So please let me know

Reply:
The problem with that is the "Windows" environment is not the main expected environment. Yes, for the desktop app, but for personal server the expected environment is Linux, so if we make a non-standard behavior, I would prefer to make it in Windows, instead of moving that to Linux. What do you think @chenilim?

Reply:
/cc @wuwinson